civihr_default_theme/scss/components/_header.scss

```diff
@@ -106,12 +104,12 @@
 
 .chr_header__logo {
   color: $light;
-  font-size: 1.6rem;
   display: inline-block;
+  font-size: 1.6rem;
 }
 
 .chr_header__nav {
-  background: #ffffff;
+  background: #fff;
```

Comment:
`$crm-white: #fff !default;`

Reply:
Thanks for the suggestion. In SSP it's called `$light` though, so using the same.